app/views/results/common/_annotations.js.erb

```diff
@@ -225,10 +237,10 @@
       corners = [bottomLeft, topLeft, topRight];
     }
 
-    var x1 = (originalImgW / 2) + corners[1][0];
-    var y1 = (originalImgH / 2) + corners[1][1];
-    var x2 = (originalImgW / 2) + corners[2][0];
-    var y2 = (originalImgH / 2) + corners[0][1];
+    var x1 = ((zoomAffectedUnrotatedImageWidth / 2) + corners[1][0]) * unzoom;
```

Comment:
I suspect the "unzoom" should happen before the translation. (In general translations should happen first/last)